lib/stripe/subscriptions/coupon.ex

```diff
@@ -27,6 +27,7 @@ defmodule Stripe.Coupon do
           livemode: boolean,
           max_redemptions: pos_integer | nil,
           metadata: Stripe.Types.metadata(),
+          name: String.t(),
```

Comment:
Looks like `name` is nullable

https://raw.githubusercontent.com/stripe/openapi/master/openapi/spec3.yaml

Reply:
👍